src/store/SwapMetaOut.h

```diff
@@ -10,14 +10,14 @@
 #define SQUID_SRC_STORE_SWAPMETAOUT_H
 
 #include "base/HardFun.h"
+#include "base/ToCpp.h"
 #include "store/forward.h"
 
 #include <memory>
 
-// TODO: Use CtoCpp1() from security/LockingPointer.h by moving that into base/ToCpp.h or similar.
 /// C++ wrapper for the legacy xmalloc()/xcalloc() deallocator
 /// \sa xfree_cppwrapper() with a slightly different (FREE-matching) signature.
-extern "C++" inline void xfree_cpp(const void * const x) { xfree(x); }
+CtoCpp1(xfree, const void *const)
```

Comment:
Personally, I would reduce reader confusion by removing the trailing `const`, but I do not insist on this change:

```suggestion
CtoCpp1(xfree, const void *)
```

If you keep that `const`, I would add a space to mimic the original spelling and similar use cases elsewhere:

```suggestion
CtoCpp1(xfree, const void * const)
```